evalml/tests/conftest.py

```diff
@@ -347,6 +347,112 @@ def fit(self, X, y):
     return MockEstimator
 
 
+@pytest.fixture
+def dummy_binary_estimator_class():
```

Comment:
Do we need this new fixture? I think we can reuse `dummy_classifier_estimator_class`

Reply:
Originally I wanted a fixture that solely supported binary and multiclass each for use cases that are solely dependent on those problem types but the tests pass with `dummy_classifier_estimator_class` anyways so I'll remove those